www/templates/survey/enketo/timeuse-inline.html

```diff
@@ -0,0 +1,33 @@
+<ion-content class="enketo-plugin overflow-scroll" translate-namespace="survey">
+
+  <div ng-if="existingSurvey">
+    <!-- If there is an existing survey, we want to ask the user whether they want to use it-->
+    <div ng-if="editSurveyAnswer === undefined" class="col" style="margin-top: 25%;" translate-namespace="survey">
+        <!-- if the user hasn't answered yet, we offer the options -->
+        <div class="row"><div class="col" style="font-weight: bold; font-size: 2em;" translate>{{'.prev-survey-found'}}</div></div>
+        <div class="row">
+            <div class="col" style="margin-right: 5px">
+                <button class="button button-block button-assertive button-outline" ng-click="ngDone()" style="border-style: solid !important; border-width: 1px !important;" id="consent-button"><span translate>{{'.use-prior-response'}}</span></button>
+            </div>
+            <div class="col" style="margin-left: 5px">
+                <button class="button button-block button-positive button-outline" ng-click="setEditSurveyAnswer(true)" style="border-style: solid !important; border-width: 1px !important;" id="consent-button"><span translate>{{'.edit-response'}}</span></button>
+            </div>
+        </div>
+        <!-- <enketo-survey-preview survey-result="existingSurvey"></enketo-survey-preview> -->
+    </div>
```

Comment:
I removed the `timeuse-inline` related code in my latest commit - since we are not using the inline implementation for timeuse, there is no reason to have it.

This also removes any need to check for existing survey result